pkgs/development/compilers/llvm/exegesis-force-bdver2.patch

```diff
@@ -0,0 +1,11 @@
+diff --git a/test/tools/llvm-exegesis/X86/uops-CMOV16rm-noreg.s b/test/tools/llvm-exegesis/X86/uops-CMOV16rm-noreg.s
+index 3fc1f31d54dc..a4c9bdd92131 100644
+--- a/test/tools/llvm-exegesis/X86/uops-CMOV16rm-noreg.s
++++ b/test/tools/llvm-exegesis/X86/uops-CMOV16rm-noreg.s
+@@ -1,5 +1,6 @@
+ # RUN: llvm-exegesis -mode=uops -opcode-name=CMOV16rm  -benchmarks-file=%t.CMOV16rm-uops.yaml
+ # RUN: FileCheck -check-prefixes=CHECK-YAML -input-file=%t.CMOV16rm-uops.yaml %s
++# RUN: sed -i 's,cpu_name:.*,cpu_name: bdver2,g' %t.CMOV16rm-uops.yaml
```

Comment:
Why is this here and not one line higher, by the way?

Reply:
No particular reason; it could be one line higher. It doesn't affect any of the lines that FileCheck is checking so it doesn't really make much of a difference.

Reply:
Ah I see. Annoying that we cannot just ask `llvm-exegesis` to target `bdver2`…

Reply:
Yeah, I filed an upstream bug; I think that was what the intent of the -mcpu=bdver2 flag in the line below was - looking at the code of llvm-exegesis, it's not actually used at all unless in benchmark mode.